include/mbedtls/cipher.h

```diff
@@ -319,6 +319,18 @@ typedef struct mbedtls_cipher_context_t
     /** CMAC-specific context. */
     mbedtls_cmac_context_t *cmac_ctx;
 #endif
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    /** Indicates whether the cipher operations should be performed
+     *  by Mbed TLS' own crypto library or an external implementation
+     *  of the PSA Crypto API.
+     *  This is unset if the cipher context is setup through
```

Comment:
This sentence is a bit messy. How about "This has a non-zero value if the cipher context is set up through mbedtls_cipher_setup_psa()."?

Reply:
I would like to be explicit about the differentiation between `mbedtls_cipher_setup_psa()` and `mbedtls_cipher_setup()`. Maybe the sentence becomes slightly easier to read when using 'established' instead of 'setup'.